File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/spring-web-codestart/java/src/main/java/org/acme/{resource.class-name}.tpl.qute.java
package org.acme;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("{resource.path}")
public class {resource.class-name} {

    @GetMapping
    public String hello() {
        return "{resource.response}";
    }
}

File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/spring-web-codestart/kotlin/src/main/kotlin/org/acme/{resource.class-name}.tpl.qute.kt
package org.acme

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;


@RestController
@RequestMapping("{resource.path}")
class {resource.class-name} {

    @GetMapping
    fun hello() = "{resource.response}"
}